### DISCUSSION TITLE
Textarea组件api、demo优化讨论

### DISCUSSION THREAD

**Discussion by xingyan95** [DevCloudFE/vue-devui#118] [General]
1. 【显示字数】demo。字数放在文本框内部的右下角更合适还是保持当前设计更合适？放在内部右下角，当文字较多时，会出现遮挡右下角文字的问题；放在文本框外面会占用额外的高度，比如在垂直布局表单中，设置每个表单项的间距固定`10px`，因为此组件字数会占用一部分高度，放在此组件后面的表单项从视觉上看，与此组件的间距会大于`10px`。
2. `id`参数没有必要吧？可以移除了。
3. `value`参数是双向绑定的，命名用`v-model`是不是更合适？
4. `autoFocus`参数命名改为`autofocus`。
5. API列表中缺少对`max-length`和`cssClass`的说明，但是在demo中却用到了。
6. `cssClass`参数设置自定义样式，需要讨论此命名是否合适？并且需要和其他组件设置自定义样式的参数命名保持一致。

**Comment by vaebe** [comment]
问题 2、3、4 经查看已不存在，cssClass 这个参数应该已经去除了。
